src/ui/KeyContent/KeyContent.jsx

```diff
@@ -8,13 +8,13 @@ KeyContent.propTypes = {
 
 function KeyContent({ children, ...rest }) {
   return (
-    <div className={`eea key-content ${rest.variant}`} {...rest}>
+    <div className="eea key-content" {...rest}>
```

Comment:
@g-stamatis To be HTML semantically correct, I think the output should be an unordered list `ul` `li`

See, for example: https://www.eea.europa.eu/publications/status-of-air-quality-in-Europe-2022